python/ray/worker.py

```diff
@@ -392,8 +392,12 @@ def run_function_on_all_workers(self, function):
             # We always run the task locally.
             function({"worker": self})
             # Check if the function has already been put into redis.
-            function_exported = self.redis_client.setnx(b"Lock:" + key, 1)
-            if not function_exported:
+            function_exported = _internal_kv_put(
```

Comment:
I don't want to flip that option for now. we need to be careful about changing to internal kv return semantics since it's dangerous to break other things. I don't want to do it here.